source/clear-linux/tutorials/aws-web/aws-web.rst

```diff
@@ -0,0 +1,264 @@
+
+   .. code-block:: console
+
+      cp Downloads/AWSClearTestKey.pem .
+
+#. Change the attributes of the :file:`AWSClearTestKey.pem` using the
+   :command:`chmod` as instructed in the dialog box shown in figure 11.
+
+   .. code-block:: console
+
+      chmod 400 AWSClearTestKey.pem
+
+#. Select the entire text highlighted in the :guilabel:`Example:` that is
+   shown in figure 11 and paste it into your terminal and add the following
+   text to the end of the command string: `-l clear`.  Press the :kbd:`Enter`
+   key to execute the command.
+
+   .. code-block:: console
+
+      ssh -i "AWSClearTestKey.pem" root@ec2-34-209-39-184.us-west-2.compute.amazonaws.com -l clear
+
+#. A message will appear on the terminal the the authenticty of the host can't
+   be established and prompts you with the message:
+
+   .. code-block:: console
+
+      The authenticity of host 'ec2-34-209-39-184.us-west-2.compute.amazonaws.com (34.209.39.184)' can't be established.
+      ECDSA key fingerprint is SHA256:LrziT5Ar66iBTfia8qmiIsrfBUm/UGam76U8bDR6yJc.
+      Are you sure you want to continue connecting (yes/no)?
+
+#. type `yes` and hit the :kbd:`Enter` key.  A warning will be printed to the
+   terminal and you will be at the command prompt of your new instance.
+
+   .. code-block:: console
+
+      Warning: Permanently added 'ec2-34-209-39-184.us-west-2.compute.amazonaws.com,34.209.39.184' (ECDSA) to the list of known hosts.
+      clear@clr-96a8565d0ca54b0c80364a1e5e7b0f88 ~ $ 
+
+Update the Clear Linux OS instance
+**********************************
+
+With the command prompt sitting in the :file:`$HOME` directory of your new
+|CL| instance, run the :command:`sudo swupd update` to update the operating
+system as shown in figure 12:
+
+.. figure:: figures/aws-web-12.png
+   :scale: 50 %
+   :alt: sudo swupd update
+
+   figure 12: :guilabel:`sudo swupd update`
+
+In this example we updated from version 18940 to 19100.
+
+Stop the Clear Linux OS instance
+********************************
+
+When you are finished using your AWS |CL| instance you need to stop it from
+the :guilabel:`Instances` dashboard to stop accruing charges while it is
+running.  With the current running instance highlighted:
+
+#. Select the :guilabel:`Actions` button to bring up a pull-down menu.
+#. Select the :guilabel:`Instance State` to expand the options
+#. Select :guilabel:`Stop` to shut down the running instance
+
+These steps are illustrated in figure 13.
+
+.. figure:: figures/aws-web-13.png
+   :scale: 50 %
+   :alt: Stop Instance
+
+   figure 13: :guilabel:`Stop Instance`
+
+#. A pop-up dialog box will appear warning you that an ephemeral storage of
+   your instance will be lost.  Select :guilabel:`Yes, Stop` to stop your |CL|
+   instance.
+
+.. figure:: figures/aws-web-14.png
+   :scale: 50 %
+   :alt: Stop Instances
+
+   figure 14: :guilabel:`Stop Instances`
+
+   That's it.  You are up and running with |CL| on AWS!
```

Comment:
Let's add a "Congratulations" :) - Congratulations! You are up and ...

Reply:
we seem to use that a lot.  I changed it but we need to have other salutations as well.